libpod/image/pull.go

```diff
@@ -176,6 +176,24 @@ func (ir *Runtime) pullGoalFromImageReference(ctx context.Context, srcRef types.
 		}, nil
 
 	case OCIArchive:
+		// imgName such as "oci-archive:/tmp/FOO.tar:localhost/foo:latest"
+		// or "oci-archive:/tmp/FOO.tar:domain.com/foo:tag1", "oci-archive:/tmp/FOO.tar"
+		imgNameSli := strings.SplitN(imgName, ":", 3)
```

Comment:
- Don’t use `imgName`, except maybe for the destination; start with the well-typed `srcRef`.
- `oci-archive:` paths can include `:` on Windows.

(Eventually this should get a `docker/archive.Reader`-like interface similar to the concept of #6811 . But that’s much more work.)